build/SourceLink.props

```diff
@@ -0,0 +1,11 @@
+<Project>
+  <PropertyGroup>
+    <SourceLinkCreate>true</SourceLinkCreate>
+    <SourceLinkNoAutoLF>true</SourceLinkNoAutoLF>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="SourceLink.Create.GitHub" Version="2.0.0-*" />
```

Comment:
Yes. It'll make sure it doesn't end up in the dependencies of the packages produced. Hopefully we get this fixed https://github.com/NuGet/Home/issues/4125. Which means this will automatically happen for packages marked as dev dependencies.